src/sentry/static/sentry/app/views/issueList/createSavedSearchButton.tsx

```diff
@@ -63,8 +71,9 @@ class CreateSavedSearchButton extends React.Component {
       });
   };
 
-  onToggle = event => {
-    const newState = {
+  onToggle = (event?: React.MouseEvent) => {
```

Comment:
this is right because we are checking if the event is ʻundefined` in the `jsx` file, but I don't think it will be undefined at some point

Reply:
hmm it is only undefined when called in the `onSubmit` function after a saved search has been created, which is why `event` has to be marked as optional.

Reply:
oh, sure ... I just saw that this is also being used in other places besides onClick :)